modules/admin-ui/src/main/webapp/scripts/modules/events/subresources/controllers/toolsController.js

```diff
@@ -169,16 +169,16 @@ angular.module('adminNg.controllers')
     $scope.player = {};
     $scope.video  = ToolsResource.get({ id: $scope.id, tool: 'editor' });
 
-    $scope.activeTransaction = false;
+    $scope.activeSubmission = false;
```

Comment:
Don't reuse the variable activeTransaction which is updated in 3 seconds polling intervals and indicates whether the event is being processed.